<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>761</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>640</width>
    <height>761</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>662</width>
    <height>761</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>OpenCv</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="8" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Speed:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="speed_label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color:rgb(85, 85, 255);</string>
        </property>
        <property name="text">
         <string>0 cm/s</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Battery:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="battery_label">
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 85, 127);</string>
        </property>
        <property name="text">
         <string>%100</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Kp Value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="kp_slider">
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="invertedAppearance">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="kp_value">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="takeOff_button">
        <property name="text">
         <string>TakeOff</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Kd Value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="kd_slider">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="kd_value">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="land_button">
        <property name="text">
         <string>Land</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="9" column="0">
     <widget class="QPushButton" name="close_button">
      <property name="text">
       <string>CLOSE</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <widget class="QLabel" name="label">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>Ki Value</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="ki_slider">
        <property name="maximum">
         <number>200</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="ki_value">
        <property name="text">
         <string>0</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="save_button">
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="4" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QRadioButton" name="fbRadio">
        <property name="text">
         <string>Forward-Backward</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="lrRadio">
        <property name="text">
         <string>Left-Right</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="udRadio">
        <property name="text">
         <string>Up-Down</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="keyboard_control">
        <property name="text">
         <string>Keyboard Control</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="manuel_button">
        <property name="text">
         <string>MANUEL CONTROLS</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>kp_slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>kp_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>402</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>kd_slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>kd_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>432</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>432</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ki_slider</sender>
   <signal>sliderMoved(int)</signal>
   <receiver>ki_value</receiver>
   <slot>setNum(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>230</x>
     <y>542</y>
    </hint>
    <hint type="destinationlabel">
     <x>410</x>
     <y>542</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
